### DISCUSSION TITLE
ability to skip auto fix

### DISCUSSION THREAD

**Discussion by rrrkren** [golangci/golangci-lint#2120] [Q&A] [answered]
we'd like to disable the auto fix that goimport applies while still keeping it as a check, is there a way to disable the auto fix for linters?

**Comment by Antonboom** [comment, marked as answer]
Hello!
Does `goimports` ignores `golangci-lint` `--fix` flag?

**Reply by rrrkren** [reply]
yeah putting this line in `.golangci.yml` fixes it for us:
```
  fix: false
```
